lib/cosmos/config/config_parser.rb

```diff
@@ -158,7 +171,10 @@ def parse_file(filename,
                    remove_quotes = true,
                    &block)
       @filename = filename
-      file = File.new(@filename, 'r')
+      # Create a temp file where we can write the ERB parsed output
+      file = Tempfile.new("parsed_#{File.basename(filename)}")
+      file.write(ERB.new(File.read(@filename)).result(binding))
+      file.rewind
```

Comment:
Not sure if we should run ERB on every file.   Perhaps only if it has a .erb extension (same as rails)?